index.html

```diff
@@ -13,234 +14,481 @@
+                <li> <a href=""><img src="Images/twitter.svg" alt=""></a></li>
+                <li> <a href=""><img src="Images/m vector.svg" alt=""></a></li>
+    
+    
+            </ul>
+    
+            <button class="about-button-desktop"><a href="">Get my resume</a></button>
         </div>
-        
-        <ul class="link">
-            <li><a href="">Html</a></li>
-            <li><a href="">css</a></li>
-            <li><a href="">javascript</a></li>
-         </ul>
+        <div class="aside-about-desktop">
 
-         <button class="button"><a href="">See Project</a></button>
-        
-    </section>
-    <section class="grid-item">
-        <img class="snap" src="Images/Snapshoot Portfolio (3).png" alt="">
-        <h2 class="multi">Multi-Post
-            Stories</h2>
-        <div class="container">
-            <div class="canopy">
-                <p>CANOPY</p>
-            </div>
-            
-            <div class="details">
-                <img src="Images/Counter.png" alt="">
-                <p>Backend Dev</p>
-            </div>
-            <div class="details">
-                <img src="Images/Counter.png" alt="">
-                <p>2015</p>
+            <div class="lang-desktop">
+                <h3>Languages</h3>
+                <img src="Images/arrow-down-sign-to-navigate.png" alt="">
             </div>
+    
+            <ul class=" basic-desktop">
+                <li class="stack-desktop"><img src="Images/Ellipse 1.png" alt="">
+                    <p>Javascript</p>
+                </li>
+                <li class="stack-desktop"><img src="Images/Ellipse 1 (1).png" alt="">
+                    <p>HTML</p>
+                </li>
+                <li class="stack-desktop"><img src="Images/Ellipse 1 (2).png" alt="">
+                    <p>CSS</p>
+                </li>
+            </ul>
+    
+            <ul class="lang-skills-desktop">
+                <li class="framework-desktop">
+                   <a href="">Frameworks</a>  <img class="image-desktop" src="Images/right-arrow.png" alt="">
+                </li>
+                <li><img class="shape-desktop" src="Images/Separator Bottom.png" alt=""></li>
+                <li class="framework-desktop">
+                   <a href="">Skills</a>  <img class="image-desktop" src="Images/right-arrow.png" alt="">
+                </li>
+                <li><img class="shape-1-desktop" src="Images/Separator Bottom.png" alt=""></li>
+            </ul>
+    
         </div>
-        <div class="content">
-            <p>
-                A daily selection of privately personalized reads; 
-                no accounts or sign-ups required.
-            </p>
+        </section>   
+    </div>
+  
+    <!-- This is the contact page -->
+    <section id="contact">
+        <div id="contact-wrapper">
+            <div id="form-wrapper">
+                <h2>Contact Me</h2>
+                <p class="wrapper">
```

Comment:
- [ ] [Optional] In the `About section` there are 2 blocks where the left one takes 40% and the right one the remaining 60% of the page's width, Also make sure that the `About section and Contact` form takes full width please implement it. 
- [ ] I would suggest you please stick to the design as much as possible. Please make your background color matches the design
- [ ] There is `br` tag after the skills please implement it.
### Your design
![his](https://user-images.githubusercontent.com/81727988/222914033-4f08f1d9-9a11-4d19-801d-d57c0c5e2018.png)
###  Figma design

![fimga](https://user-images.githubusercontent.com/81727988/222914053-888e9236-a361-4afe-b080-b2850dcd3947.png)